### PR TITLE
CE-CC-add-warning-allow-tigera-tier

### PR DESCRIPTION
**Pull request by bmckercher123**
### Add warning about allow-tigera tier

- Two docs: Get started with tiered network policy, and Change allow-tigera behavior
- For CE, 3.17, 3.16, 3.15
- For CC, 3.16, 3.15

### CODE REVIEW

**Review by tmjd — DISMISSED** [review]
LGTM

File: calico-cloud/network-policy/policy-tiers/allow-tigera.mdx [hunk 1]
```diff
@@ -1,9 +1,14 @@
 ---
-description: Understand the purpose of the allow-tigera tier and how to change its behavior.
+description: Understand how to change the behavior of the allow-tigera tier.
 ---
 
 # Change allow-tigera tier behavior
 
+:::warning
```
Comment (ctauchen) [inline calico-cloud/network-policy/policy-tiers/allow-tigera.mdx]: Small fiddly thing, but best to have a space between the admonition fence and the content.

File: calico-cloud/network-policy/policy-tiers/allow-tigera.mdx [hunk 2]
```diff
@@ -1,9 +1,14 @@
...
+The `allow-tigera` tier contains policies that secure {{prodname}} components and is critical to cluster integrity. It is controlled by the Tigera Operator, and policies in the tier should not be edited, and the tier should not be moved. Although you can change the behavior of allow-tigera using adjacent tiers, you can inadvertantly break critical cluster traffic. We highly recommend that you work with Support to implement changes around `allow-tigera` to prevent service disruption.
```
Comment (ctauchen) [inline calico-cloud/network-policy/policy-tiers/allow-tigera.mdx]: s/inadertantly/inadvertently

File: calico-cloud/network-policy/policy-tiers/tiered-policy.mdx [hunk 3]
```diff
@@ -64,16 +64,17 @@ The default tier is where:
 
 ## System tiers
 
-System tiers are added during installation and are hidden by default.
+System tiers are added during installation and are hidden by default. Create all of your tiers after `allow-tigera` and `tigera-security`.
```
Comment (ctauchen) [inline calico-cloud/network-policy/policy-tiers/tiered-policy.mdx]: I'm not sure what this means. The new sentence makes it sounds like you can create allow-tigera and tigera-security, and that you must do this first. But the preceding sentence says this happens at installation. 

Can you create tiers before allow-tigera is created?

File: calico-cloud/network-policy/policy-tiers/tiered-policy.mdx [hunk 4]
```diff
@@ -64,16 +64,17 @@ The default tier is where:
...
 
-- **allow-tigera** - contains component policies
-- **tigera-security** - contains threat protection policies
+- **allow-tigera** tier contains policies to secure {{prodname}} components and are controlled by the Tigera Operator. These policies should not be edited, and the tier should not be moved. Inadvertent changes are automatically reverted by the Operator to ensure your cluster is always protected.
+
+   :::warning
 
-:::note
+   Although it is possible to change the behavior of the `allow-tigera` using adjacent tiers, it is not a trivial task. You can break critical cluster traffic and impact the operation of {{prodname}}. To prevent loss of cluster services, see [Change allow-tigera tier behavior](allow-tigera.mdx), and contact Support for help.
```
Comment (ctauchen) [inline calico-cloud/network-policy/policy-tiers/tiered-policy.mdx:73]: "it is not a trivial task." makes it sounds merely difficult. Perhaps better to describe it as risky or dangerous? Just a thought.

**Review by ctauchen — DISMISSED** [review]
A few suggestions and comments. Otherwise LGTM.

File: calico-cloud/network-policy/policy-tiers/allow-tigera.mdx (hunk 1 above)
Comment (bmckercher123) [reply calico-cloud/network-policy/policy-tiers/allow-tigera.mdx]: Fixed.

File: calico-cloud/network-policy/policy-tiers/allow-tigera.mdx (hunk 2 above)
Comment (bmckercher123) [reply calico-cloud/network-policy/policy-tiers/allow-tigera.mdx]: Fixed.

File: calico-cloud/network-policy/policy-tiers/tiered-policy.mdx (hunk 4 above)
Comment (bmckercher123) [reply calico-cloud/network-policy/policy-tiers/tiered-policy.mdx:73]: It is both difficult in terms of the policy, and risky to get it right.

File: calico-cloud/network-policy/policy-tiers/tiered-policy.mdx (hunk 3 above)
Comment (bmckercher123) [reply calico-cloud/network-policy/policy-tiers/tiered-policy.mdx]: The guidance for users is to create their tiers after the system tiers (in terms of left to right tier ordering). Tweaked text.

**Pull request merged**